test/e2e/main_test.go

```diff
@@ -59,14 +49,15 @@ func TestMain(m *testing.M) {
 	os.Exit(m.Run())
 }
 
-func TestReportingE2E(t *testing.T) {
+func TestReportingProducesData(t *testing.T) {
 	hourlySchedule := &meteringv1alpha1.ReportSchedule{
 		Period: meteringv1alpha1.ReportPeriodHourly,
 	}
 
 	queries := []struct {
-		queryName string
-		skip      bool
+		queryName   string
+		skip        bool
+		nonParallel bool
```

Comment:
`serial`? not sure that's better, but it's at least not going to lead to `!query.nonParallel`